client/my-sites/customer-home/cards/tasks/connect-accounts.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import QueryPublicizeConnections from 'components/data/query-publicize-connections';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ConnectAccountsTask = ( { skipTask, siteId, siteSlug } ) => {
+	const translate = useTranslate();
+
+	return (
+		<ActionPanel className="tasks connect-accounts">
+			<QueryPublicizeConnections selectedSite />
+			<ActionPanelBody>
+				{ ! isMobile() && (
+					<ActionPanelFigure align="right">
+						<img src="/calypso/images/stats/tasks/social-links.svg" alt="" />
+					</ActionPanelFigure>
+				) }
+				<div className="tasks__timing">
+					<Gridicon icon="time" size={ 18 } />
+					<p>{ translate( '3 minutes' ) }</p>
+				</div>
+				<ActionPanelTitle>{ translate( 'Drive traffic to your site' ) }</ActionPanelTitle>
+				<p className="tasks__description">
+					{ translate(
+						'Integrate your site with social media to automatically post your content and drive traffic to your site.'
+					) }
+				</p>
+				<ActionPanelCta>
+					<Button
+						className="tasks__action"
+						primary
+						href={
+							isMobile()
+								? `/marketing/connections/${ siteSlug }`
+								: `/marketing/connections/${ siteSlug }?tour=marketingConnectionsTour`
+						}
+					>
+						{ translate( 'Connect accounts' ) }
+					</Button>
+					<Button
+						className="tasks__action-skip is-link"
+						onClick={ () => {
+							skipTask( 'homeTaskConnectAccounts-' + siteId, true );
```

Comment:
Dismissed card preferences are [prefixed with `dismissible-card`](https://github.com/Automattic/wp-calypso/blob/0832cae4288d3d9aa10fec71ed280f125f0c372a/client/blocks/dismissible-card/index.jsx#L83) when using the `DismissableCard` component, so we can maintain the same behavior too.

That's important because the API automatically excludes a card from the Home layout if there is preference name that matches the `dismissable-card-${ cardIdentifier }-${ siteId }` pattern. There is no card identifier for this card in the API yet, but we can assume it will be `home-task-connect-accounts`, so we could use it here:

```suggestion
							skipTask( 'dismissal-card-home-task-connect-accounts-' + siteId, true );
```

Also, note that since we're going to store a `true` value always, we can abstract a little bit the logic, but this is completely optional.

```
const mapDispatchToProps = {
  skipTask: siteId => savePreference( `dismissal-card-home-task-connect-accounts-${ siteId }`),
};

const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
  return {
    ...stateProps,
    skipTask: () => dispatchProps.skipTask( stateProps.siteId ),
   ...ownProps,
};

export default connect( mapStateToProps, mapDispatchToProps, mergeProps )( ConnectAccountsTask );
```

Then, `onClick` should simply call `skipTask`:

```
onClick={ skipTask }
```

Reply:
Thank you! I've updated the preference name as suggested, but with the other logic you mentioned, it appears to be causing an issue that I'm unsure how to fix: `'import' and 'export' may only appear at the top level (90:0)`

<img width="766" alt="Screenshot 2020-04-13 at 12 14 02" src="https://user-images.githubusercontent.com/43215253/79116714-24361a00-7d81-11ea-964b-c160f6d79804.png">

Reply:
My example code was untested, and I noted I missed a closing curly bracket in `mergeProps`. 

```
const mergeProps = ( stateProps, dispatchProps, ownProps ) => {
  return {
    ...stateProps,
    skipTask: () => dispatchProps.skipTask( stateProps.siteId ),
    ...ownProps,
  }
};
```

Does that solve the issue?

Reply:
Oops! Yes, that works, but I'm assuming that the preference should still be saved just like before? 

<img width="421" alt="Screenshot 2020-04-13 at 13 39 02" src="https://user-images.githubusercontent.com/43215253/79120995-2a7dc380-7d8c-11ea-9ee1-be1be4fb17f2.png">

I don't think that I'm able to get that to work.

Reply:
Yeah, sorry, there is a couple of additional errors on my example code 😅:
- Dismissed cards uses the `dismissible-card` prefix, not `dismissal-card`.
- We still need to pass the `true` value when calling to `savePreference`.

Reply:
Thank you! I've committed this approach now. :)